processing/src/main/java/io/druid/segment/BitmapOffset.java

```diff
@@ -40,6 +40,21 @@
 
   private volatile int val;
 
+  public static IntIterator getReverseBitmapOffsetIterator(ImmutableBitmap bitmapIndex)
+  {
+    ImmutableBitmap roaringBitmap = bitmapIndex;
+    if (!(bitmapIndex instanceof WrappedImmutableRoaringBitmap)) {
+      final BitmapFactory factory = RoaringBitmapSerdeFactory.bitmapFactory;
```

Comment:
I'm not planning on changing the default bitmap to roaring with this PR at least.

if anyone knows of a faster way to get a reverse bitmap iterator, please let me know